packages/@react-aria/slider/src/useSliderThumb.ts

```diff
@@ -70,15 +70,29 @@ export function useSliderThumb(
     reverse: false,
     orientation: 'horizontal',
     onDrag: (dragging) => {
+      if (state.isTrackDragging()) {
```

Comment:
Would an alternative way of handling this be for `useDrag1D` to call `stopPropagation` on any events it handles? That way, if the thumb is a child of the track, the track would never receive the event for drags started on the thumb. We tend to do this in most places we handle other events already (e.g. `usePress`). Once an event is handled, it's pretty rare that we'd also want to handle it on a parent element, and that would avoid needing to track this state here. Probably would also fix other issues that may arise when using `useDrag1D` in other components as well.

Reply:
Ah, good point; will give that a shot!